src/vald/v1/rpc/errdetails/error_details_pb2.py

```
@@ -0,0 +1,68 @@
+    5,
+    28,
+    3,
+    '',
+    'vald/v1/rpc/errdetails/error_details.proto'
+)
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.protobuf import duration_pb2 as google_dot_protobuf_dot_duration__pb2
```

Ruff

`google.protobuf.duration_pb2` imported but unused

Remove unused import: `google.protobuf.duration_pb2`

(F401)